### ISSUE TITLE
solr: 4 services is overkill

### ISSUE THREAD

**Issue by wdavidw** [ryba-io/ryba#90]
3 service definitions in solr + 1 inside ranger (configured by ranger admin), could we strip it down to 1 or 2 ? There is no need to support all the deployment options of a component.

**Comment by leopaul36** [comment]
We could remove `standalone` as its main use case is now being supported by the embedded version in Ranger. `cloud_docker` is working well and I don't know the use cases for `cloud` so I will let @lucasbak decide if we can remove it

**Comment by lucasbak** [comment]
Yes indeed you could remove standalone as its purpose was only for ranger. Now we should not remove cloud as it has the same use cases Than cloud_docker but being more simple to install as it does not run in docker. It's no hard to maintain

**Comment by wdavidw** [comment]
OK, the idea would be to remove solr/standalone and ranger/solr ?

**Comment by IDerr** [comment]
I think we should remove cloud_docker but keep the others. But @lucasbak will give his words :D

**Comment by lucasbak** [comment]
Not at all only remove solr/standalone

**Comment by wdavidw** [comment]
Seems like "cloud_docker" is the [default](https://github.com/ryba-io/ryba/blame/master/ranger/admin/configure.coffee.md#L185)

**Comment by wdavidw** [comment]
Note, this is an open question, if they all make sense then fine

**Comment by lucasbak** [comment]
Yep it's the default but we can remove standalone
